content/performance/docs/binpack.md

```diff
@@ -0,0 +1,421 @@
+                  resources:
+                      - name: cpu
+                        weight: 1
+                      - name: memory
+                        weight: 1
+                  type: MostAllocated
+            name: NodeResourcesFit
+        plugins:
+          score:
+              enabled:
+                  - name: NodeResourcesFit
+                    weight: 1
+              disabled:
+                  - name: "*"
+          multiPoint:
+              enabled:
+                  - name: NodeResourcesFit
+                    weight: 1
+        schedulerName: my-scheduler
+    leaderElection:
+      leaderElect: true
+      resourceNamespace: kube-system
+      resourceName: my-scheduler
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: my-scheduler
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      component: scheduler
+      tier: control-plane
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: karpenter.sh/nodepool
+                operator: DoesNotExist
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchLabels:
+                component: scheduler
+                tier: control-plane
+            topologyKey: kubernetes.io/hostname
+      serviceAccountName: my-scheduler
+      containers:
+      - command:
+        - /usr/local/bin/kube-scheduler
+        - --bind-address=0.0.0.0
+        - --config=/etc/kubernetes/my-scheduler/my-scheduler-config.yaml
+        - --v=5
+        image: public.ecr.aws/eks-distro/kubernetes/kube-scheduler:v1.28.11-eks-1-28-latest
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 10259
+            scheme: HTTPS
+          initialDelaySeconds: 15
+        name: kube-second-scheduler
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 10259
+            scheme: HTTPS
+        resources:
+          requests:
+            cpu: '1'
+        securityContext:
+          privileged: false
+        volumeMounts:
+          - name: config-volume
+            mountPath: /etc/kubernetes/my-scheduler
+      hostNetwork: false
+      hostPID: false
+      volumes:
+        - name: config-volume
+          configMap:
+               name: my-scheduler-config
+EOF
+```
+
+
+## Validate the Custom Scheduler
+
+- **Step1:** Launch the node viewer in a terminal:
+```bash
+eks-node-viewer --resources cpu,memory
+```
+
+- **Step 2:** Add the custom scheduler to an EMR on EKS job via Pod Templates. For exmaple:
+
+**executor-pod-template.yaml (line #3)**
```

Comment:
We can simplify and show that its also possible by just passing the scheduler in the spark conf, using `spark.kubernetes.scheduler.name`, without using the a pod template.

Reply:
added the extra example. see the 
[improve binpack](https://github.com/aws/aws-emr-containers-best-practices/pull/57/commits/55123ad158cd508fbc36ab7faedba6ef833598d4)